test/controllers/user.test.js

```diff
@@ -66,3 +67,52 @@ describe('usersController.signUp', () => {
       });
   });
 });
+
+describe('usersController.signIn', () => {
+  it('Log in with previously created user', () => {
+    userAttributes.password = 'password1923';
+    return request
+      .post('/users')
```

Comment:
Have a look at this: https://www.npmjs.com/package/factory-girl for generating users. It won't add much now but it really helps when you need to generate several models.

Reply:
I prefer let this like one method than register a specific user, in the next PR i'll add and use that package.